core/slice.go

```diff
@@ -767,6 +767,10 @@ func (sl *Slice) init() error {
 			genesisTermini.SetDomTerminiAtIndex(genesisHash, i)
 		}
 
+		rawdb.WriteBetas(sl.sliceDb, genesisHash, big.NewFloat(0.5), big.NewFloat(0.00083222))
```

Comment:
Are these temp numbers? Should they be in protocol_params?

Reply:
we will put real values in after observing the market, until then this will do as a initializer values